receiving_collection.json

```diff
@@ -50,14 +51,14 @@
         "additionalProperties": false
```

Comment:
Please add here:
```json
        "required": [
          "po_line_id"
        ],
```
In this case the jaxrs-code-generator will be able to add `@NotNull` annotation in the model for the corresponding property.